effect/README.md

```diff
@@ -39,7 +39,7 @@ import scalaz.effect.{IO, SafeApp}
 import scalaz.effect.console._
 
 object MyApp extends SafeApp {
-  def run(args: IList[String]): IO[Unit] =
+  def run(args: IList[String]): IO[E, Unit] =
```

Comment:
Where's type variable `E` bound? Or is it a top-level type named `E`? Or a _meta_variable `E` which isn't introduced yet? Your examples seem to use `Error` instead.